include/roboteam_utils/Circle.h

```diff
@@ -0,0 +1,97 @@
+//
+// Created by emiel on 24-02-20.
+//
+
+#ifndef RTT_CIRCLE_H
+#define RTT_CIRCLE_H
+#include "Rectangle.h"
+#include "Vector2.h"
+
+namespace rtt {
+/**
+ * @brief Represents a circle by storing the center and the radius
+ * @date 24-02-2020
+ * @author Emiel Steerneman
+ */
+class Circle {
+   public:
+    Vector2 center;
+    double radius;
+
+    /**
+     * @brief Constructs a circle with a default center and radius
+     */
+    Circle();
```

Comment:
Proposal: make it the unit circle : Vector2(0,0) and radius 1.0

Reply:
Sounds good to me